TestLogin/TestLogin/TestLogin/DisplayMessages.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestLogin
+{
+    internal interface DisplayMessages
+    {
+        Task ShowAsync(string message);
+    }
+    public class ServiceMessage : DisplayMessages
+    {
+        public async Task ShowAsync(string message)
+        {
+            await App.Current.MainPage.DisplayAlert("YourApp", message, "Ok");
+            throw new NotImplementedException();
```

Comment:
Is this Tested? Why u throw a exception?

Reply:
Sorry. I forgot to Delete DisplayMessages.cs It is not serving a purpose.

Reply:
Class Removed